README.md

```diff
@@ -158,7 +158,8 @@ Run `lerna --help` to see all available commands and options.
       "npmClientArgs": ["--no-package-lock"]
     }
   },
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "useWorkspaces": false
```

Comment:
Why would you explicitly configure a value to `false`? It's confusing the issue. I agree `useWorkspaces` is poorly documented, but it needs more clarity than this.

Reply:
I don't get how 'false' is more or less confusing than 'true', but whatever suits the project best works for me.

Reply:
I mean, `false` is the default. I think people understand how to set boolean properties in JSON files?

What I meant, more specifically, is a separate section that describes the changes between multiple files in the monorepo. It's not just a change to the `lerna.json`, after all, when using Yarn (or npm v7) workspaces.